great_expectations/render/renderer_configuration.py

```diff
@@ -479,7 +477,6 @@ def _parse_row_condition_str(row_condition_str: str) -> str:
             .replace(" or ", " OR ")
             .replace("~", " NOT ")
             .replace(" not ", " NOT ")
-            .replace("==", " is ")
```

Comment:
Decided we should handle this logic in GX Cloud since there are rendering substitutions like this that need to be done and are strictly GX Cloud configuration options.